### ISSUE TITLE
TypeError in /tracks/python/exercises/black-jack

### ISSUE THREAD

**Issue by iHiD** [exercism/website#1348]
## Error in v3 (client side)

**TypeError** in **/tracks/python/exercises/black-jack**
Cannot read property 'filename' of undefined

[View on Bugsnag](https://app.bugsnag.com/exercism/v3-client-side/errors/6108379fcb99b80007e3a590?event_id=6108379f007fa8ce9b830000&i=gh&m=ci)

## Stacktrace

    app/javascript/components/modals/exercise-update-modal/ExerciseUpdateForm.tsx:27 - ExerciseUpdateForm

[View full stacktrace](https://app.bugsnag.com/exercism/v3-client-side/errors/6108379fcb99b80007e3a590?event_id=6108379f007fa8ce9b830000&i=gh&m=ci)

*Created automatically via Bugsnag*

**Comment by kntsoriano** [comment]
@iHiD This returns a diff with an empty files array. What should we do about this? Are we ensured that a diff would always have files?

**Comment by iHiD** [comment]
Yes. We **should** be after ETL. If we're not, I'll want to know about it and fix it then. So I'm happy for this to be closed after ETL and if it reopens I'll fix it on a case-by-case basis.

However, we might want to add a guard that:
- Raises a bugsnag in **Rails** if it's empty
- Guards against empty in the JS (and shows the modal with some text that says "Sorry - it seems that we can't work out what needs updating for this exercise. We've been alerted and will have a look." rather than raising a client-side exception.

So could you open a PR that does that pls?

**Comment by iHiD** [comment]
An error linked to this issue has been reopened in Bugsnag
[**TypeError** in **/tracks/python/exercises/two-fer**](https://app.bugsnag.com/exercism/v3-client-side/errors/6108379fcb99b80007e3a590?event_id=612d92a6007f834d1f1b0000&i=gh&m=ra)
